toolchains/arm-linux-gnueabi.cmake

```diff
@@ -0,0 +1,23 @@
+# Copyright (C) 2022 Toitware ApS.
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; version
+# 2.1 only.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# The license can be found in the file `LICENSE` in the top level
+# directory of this repository.
+
+set(ARM_TARGET "arm-linux-gnueabi")
+
+set(ARM_CPU_FLAGS "-marm -march=armv5te -mtune=arm926ej-s -msoft-float -mfloat-abi=soft")
```

Comment:
It feels a bit weird that arm-linux-gnueabi implies arvm5te and specific tuning.

Reply:
Agree.
That said, we had to make a similar decision for the raspberry pi.

I think users will need to adapt these flags anyway.
Open for suggestions, but would leave it as is for now.